packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -181,15 +182,15 @@ function updateRipple (el: HTMLElement, binding: VNodeDirective, wasEnabled: boo
     el._ripple.circle = value.circle
   }
   if (enabled && !wasEnabled) {
-    el.addEventListener('touchstart', rippleShow, { passive: true })
-    el.addEventListener('touchend', rippleHide, { passive: true })
+    addPassiveEventListener(el, 'touchstart', rippleShow as any)
```

Comment:
as any?

These also aren't actually passive now.

Reply:
Couldn't figure out the typing. As far as the passive issue, that's resolved.

Reply:
I've removed all framework modifications.